tests/unit/helpers/test_dashboard_publisher.py

```diff
@@ -0,0 +1,45 @@
+from pathlib import Path
+from unittest.mock import create_autospec
+
+from databricks.labs.blueprint.installation import MockInstallation
+from databricks.sdk import WorkspaceClient
+from databricks.sdk.service.workspace import ObjectInfo, ObjectType
+
+from databricks.labs.remorph.helpers.dashboard_publisher import DashboardPublisher
+
+
+def test_create_dashboard():
+    installation = MockInstallation(is_global=False)
+    workspace_client = create_autospec(WorkspaceClient)
+    dashboard_file = Path(__file__).parent / Path('../../resources/Remorph-Reconciliation.lvdash.json')
```

Comment:
As per this [link](https://stackoverflow.com/questions/6028000/how-to-read-a-static-file-from-inside-a-python-package), we shouldn't use `__file__`. 

```
Bad ways to avoid:
Bad way #1: using relative paths from a source file
This was previously described in the accepted answer. At best, it looks something like this:

from pathlib import Path

resource_path = Path(__file__).parent / "templates"
data = resource_path.joinpath("temp_file").read_bytes()
What's wrong with that? The assumption that you have files and subdirectories available is not correct. This approach doesn't work if executing code which is packed in a zip or a wheel, and it may be entirely out of the user's control whether or not your package gets extracted to a filesystem at all.

```

Reply:
This is ok for test cases. all are valid if this is being used inside our core lib.

Reply:
Done. Used importlib.